pyas2/models.py

```diff
@@ -460,6 +463,13 @@ def status_icon(self):
 
     def send_message(self, header, payload):
         """Send the message to the partner"""
+
+        if self.organization_id and self.partner_id:
+            self.organization = self.organization or Organization.objects.get(
```

Comment:
reason for this change?

Reply:
I have noted that under certain circumstances (i.e. update_or_create) will not keep the related object inside the model. So, in case that this is happening, then we pull it once more, but we are not pulling it, when it exists already, thus not executing another query.